src/EnergyPlus/UnitarySystem.cc

```diff
@@ -515,6 +518,9 @@ namespace UnitarySystems {
                 if (this->m_FanExists && (this->m_CoolCoilExists && (this->m_HeatCoilExists || this->m_SuppCoilExists)))
                     state.dataAirLoop->AirLoopControlInfo(AirLoopNum).UnitarySys = true;
                 state.dataAirLoop->AirLoopControlInfo(AirLoopNum).UnitarySysSimulating = true;
+                if (this->m_CoolingCoilType_Num == DataHVACGlobals::CoilDX_CoolingTwoSpeed) {
+                    DXCoils::SetCoilSystemCoolingData(state, this->m_CoolingCoilName, this->Name);
+                }
```

Comment:
I can't recall why I put this here and not in getInput. Probably a late night. This just sets the CoilSystem name in state.dataDXCoils->DXCoil(WhichCoil).CoilSystemName which is then used in SimAirServingZones. Not typical.